internal/events/get.go

```diff
@@ -47,15 +58,12 @@ func get(
 	_ command.GlobalFlags,
 	services *services.Services,
 ) (command.Result, error) {
-	name := args[0]
-	start := args[1] // block height range start
-	end := ""        // block height range end
-
-	if len(args) == 3 {
-		end = args[2]
+	start, end, err := services.Events.CalculateStartEnd(eventsFlags.Start, eventsFlags.End, eventsFlags.Last)
```

Comment:
Argument parsing should be done in the command itself, in this current implementation it exposes this API to anyone using the flowkit as a lib and I don't think it is necessary, because when using the lib programmatically you don't necessarily use user input.

Reply:
When calling this code from literaly any other place except the cli you would use actual block heights here. Having the services api know about -10 as meaning the 10 latest blocks is not very logical if you ask me. I would prefer parsing it in the cli layer and having a service layer that gets startBlock/endBlocks as uints as data.

Reply:
We could move the method into the cli layer and just use the service layers block service to fetch the latest block height.

Reply:
Yeah, I agree that having the service layer know about last X blocks is not ideal, but then the API should be changed to the proposed one `services.Events.Get(name, start, end)` and the start/end can be calculated accordingly in the command layer as you said. I think that having that function for the block calculation in the events API is not so great because of the same reasons you just mentioned - not many devs will use last X blocks when using the SDK.

Reply:
inlined the argument parsing into the cli layer and removed the method from the service layer.